discuss/getting_started.rst

```diff
@@ -4,187 +4,13 @@
 Discuss
 =======
 
-Overview
```

Comment:
Actually getting_started.rst should be removed and a different rst should be created for each section (To have section titles)